content/guides/concepts/dnslink.md

```diff
@@ -43,3 +51,24 @@ Will get you this block:
 ```
 /ipfs/QmYNQJoKGNHTpPxCBPh9KkDpaExgd2duMa3aF6ytMpHdao/media/
 ```
+
+### Publish via Subdomain
+
+It is possible to publish DNSLink using a special subdomain named `_dnslink`.
+
+Example: `docs.ipfs.io` may not not have a TXT record, but the page still loads
+because TXT record exists for `_dnslink.docs.ipfs.io`:
+
+```
+dig +noall +answer TXT _dnslink.docs.ipfs.io
+_dnslink.docs.ipfs.io.  34  IN  TXT "dnslink=/ipfs/QmeveuwF5wWBSgUXLG6p1oxF3GKkgjEnhA6AAwHUoVsx6E"
+```
+
+<!-- Not sure if IPNS is fast enough to mention here
```

Comment:
Does it resolve pretty reliably before your browser times out yet? That might be a better way to decide whether it’s worth discussing.

Reply:
AFAIK it is hit or miss right now. Tthat is why I commented out this section, so it won't get rendered.
We can revisit it when IPNS improves enough.

Reply:
Makes sense!